src/Sylius/Bundle/CoreBundle/SyliusCoreBundle.php

```diff
@@ -47,6 +48,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new LazyCacheWarmupPass());
         $container->addCompilerPass(new SitemapProviderPass());
         $container->addCompilerPass(new RegisterTaxCalculationStrategiesPass());
+        $container->addCompilerPass(new RegisterOrderProcessorsPass());
```

Comment:
`CompositeOrderProcessor` is located in Core component and its coupled to `Sylius\Component\Core\Model\OrderInterface` and `Sylius\Component\Core\OrderProcessing\OrderProcessorInterface`.

Moving the compiler pass to `OrderBundle` will couple it to `CoreBundle` if I am not mistaken